src/bitmessagecli.py

```diff
@@ -30,32 +30,252 @@
 keysName = 'keys.dat'
 keysPath = 'keys.dat'
 usrPrompt = 0  # 0 = First Start, 1 = prompt, 2 = no prompt if the program is starting up
-knownAddresses = dict()
+
+knownAddresses = {'addresses':[]}
+
+import inspect
+import re
+
+cmdstr = 'buildknownaddresses'
+
+from collections import OrderedDict
+# menu by this order
+cmdTbl = OrderedDict({})
```

Comment:
It also can be written as
```python
cmdTbl = OrderedDict(
    Command='Description',
    s1='-',
...
)
```

Reply:
@g1itch order changed.

Reply:
You are right, it might be
```python
cmdTbl = OrderedDict([
    ('Command', 'Description'),
    ('s1', '-'),
...
])
```

Any way it was just example. It is not necessary to change anything here.